### PR TITLE
Implemented esim api's and send grid api

### PR DESCRIPTION
**Pull request by GetFridayDevlopment**

### CODE REVIEW

File: dynamo_client.py [hunk 1]
```diff
@@ -1,44 +1,120 @@
...
-            'source_name': order.source_name,
-            'source_order_id': order.source_order_id,
-            'customer_id': customer.customer_id,
-            'total_price': order.price,
-            'order_items': order.order_items,
-            'upserted_at': str(datetime.now())
-        })
-
+                    'order_id':order.id,
+                    'source_name':order.souce_name,
```
Comment (zaine2003) [inline dynamo_client.py]: is order.souce_name correct?

File: esim_go_client.py [hunk 2]
```diff
@@ -1,28 +1,88 @@
-import urllib3
+import requests
```
Comment (zaine2003) [inline esim_go_client.py]: https://stackoverflow.com/questions/40741282/cannot-use-requests-module-on-aws-lambda you might need to follow this to get `requests` working

File: lambda_function.py [hunk 3]
```diff
@@ -31,3 +30,54 @@ def lambda_handler(event, context):
...
+    
+
+    esim_details = dynamo_client.get_esim_from_db(order_id)
+    if not esim_details:
+        print("Failed to get esim_details from db :" + esim_details)
+        return
+  
+
+    if qr_codes:
+        email_client.send_email_with_qr_code('sivasankar.selva@gmail.com', 'QR Code Email', qr_codes, esim_details)
```
Comment (zaine2003) [inline lambda_function.py]: have we got a ticket to swap this out when we need to?

File: dynamo_client.py (hunk 1 above)
Comment (GetFridayDevlopment) [reply dynamo_client.py]: yes, @zaine2003 is not correct.I will change and push.

File: esim_go_client.py (hunk 2 above)
Comment (GetFridayDevlopment) [reply esim_go_client.py]: @zaine2003 we can use urllib3

File: lambda_function.py (hunk 3 above)
Comment (GetFridayDevlopment) [reply lambda_function.py]: @zaine2003 you mean the email address right? testing I added we can change to customer email here.

**Review by zaine2003 — APPROVED** [review]

**Pull request merged**